docker/transformers-all-latest-gpu/Dockerfile

```diff
@@ -51,7 +51,8 @@ RUN python3 -m pip install --no-cache-dir git+https://github.com/huggingface/acc
 # Add bitsandbytes for mixed int8 testing
 RUN python3 -m pip install --no-cache-dir bitsandbytes
 
-RUN python3 -m pip install --no-cache-dir decord
+# For video model testing
+RUN python3 -m pip install --no-cache-dir decord av==9.2.0
```

Comment:
Not removing decord yet as it's used in some other areas e.g. [video classification pipeline](https://github.com/huggingface/transformers/blob/92dfceb12446404fe547b1271ecaf86d2498546b/src/transformers/pipelines/video_classification.py#L12)